src/columns/nano-id.ts

```diff
@@ -0,0 +1,24 @@
+import * as glide from "../glide";
+import { nanoid, customAlphabet } from "../../node_modules/nanoid/async";
+
+export default glide
+    .columnNamed("Nano Id")
+    .withCategory("Text")
+    .withDescription(`A tiny (108 bytes), secure, URL-friendly, unique string ID generator for JavaScript`)
+    .withNumberParam("num", "Length")
+    .withStringParam("alphabet", "Custom Alphabet")
+    .withAuthor("Andrey Sitnik", "https://github.com/ai/nanoid")
+    .withFailingTest({ num: 10 }, "test")
```

Comment:
You should have real tests.

Reply:
With the current implementation, I do not see a way to make a non-failing test. I suppose this would only work if the Custom Random Bytes Generator is stable when a seed is provided.